python/ray/autoscaler/node_provider.py

```diff
@@ -141,6 +141,22 @@ def terminate_nodes(self, node_ids: List[str]) -> Optional[Dict[str, Any]]:
             self.terminate_node(node_id)
         return None
 
+    @property
+    def max_terminate_nodes(self) -> Optional[int]:
+        """The maximum number of nodes which can be terminated in one single
+        API request. By default, this is "None", which means that the node
+        provider's underlying API allows infinite requests to be terminated
+        with one request.
+
+        For example, AWS only allows 1000 nodes to be terminated
+        at once; to terminate more, we must issue multiple separate API
+        requests. If the limit is infinity, then simply set this to None.
+
+        This may be overridden. The value may be useful when overriding the
+        "terminate_nodes" method.
+        """
+        return None
```

Comment:
why do we need this? 
It does not seem to be needed now for the AWSNodeProvider. It does not seem to be doing anything anyway.